sympy/printing/llvmjitcode.py

```diff
@@ -0,0 +1,175 @@
+from __future__ import print_function, division
+
+'''
+Use llvmlite to create executable functions from Sympy expressions
+
+  Prerequisites
+  -------------
+  LLVM - http://llvm.org/
+  llvmlite - https://github.com/numba/llvmlite
+
+Examples:
+
+Scalar expression
+-----------------
+  import sympy.printing.llvmlitecode as g
+
+  a = Symbol('a')
+  e = a*a + a + 1
+  e1 = g.get_jit_callable(e, args=[a])
+  print(e1(1.1), e.subs('a',1.1))
```

Comment:
This should follow our docstring style so that it get's tested, i.e. use `>>>` to proceed input lines.